src/scenes/Event/scenes/topic.js

```diff
@@ -349,7 +344,7 @@ export default class TopicPage extends React.Component {
       <TableRow key={index}>
         <TableCell padding="dense">
           <div>{withdrawableAddress.address}</div>
-          { id && message && <div className={warningTypeClass}>{intl.formatMessage({ id, message })}</div> }
+          {id && message && <EventWarning id={id} message={message} className={warningTypeClass} />}
```

Comment:
Don't believe this is a one-to-one replacement. The text for this doesn't use the border around the `EventWarning`.

Reply:
I will take a looksy